src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -3668,6 +3668,9 @@ private CheckValueStatus TryChangeTypeSpecial(ref object value)
             return CheckValueStatus.Success;
         }
 
+        // Remove this once Mono support for function pointers (#71095) is addressed.
+        internal bool IsIntPtrBasedFunctionPointer => IsFunctionPointer;
```

Comment:
What are the failing tests without this workaround (outside the tests that are disabled against adding function pointer support to Mono)?

Reply:
I guess this workaround can be reverted now that the support for function pointers in Mono was merged.